modules/tinymce/src/core/main/ts/init/InitIframe.ts

```diff
@@ -88,7 +89,32 @@ const init = (editor: Editor, boxInfo: BoxInfo): void => {
   editor.getElement().style.display = 'none';
   DOM.setAttrib(editor.id, 'aria-hidden', 'true');
 
-  InitContentBody.initContentBody(editor);
+  // Restore visibility on target element
```

Comment:
This makes the init function rather fat. Could you make this a local function in this module. I think it still separate thing since it deals with the inside of the iframe. The rest of the init creates the iframe and sets aria hidden on the target element so different things. Maybe rename it to say `setupIframeBody` or something?

- [ ] Consider keeping the function but making it an internal function of the InitIframe module

Reply:
Ah yes I just inlined the function call but that's a good idea, thanks